CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxVisitorFile.swift

```diff
@@ -18,12 +18,12 @@ import Utils
 let syntaxVisitorFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
   DeclSyntax(
     """
-    /// The enum describes how the SyntaxVistor should continue after visiting
+    /// The enum describes how the SyntaxVisitor should continue after visiting
```

Comment:
Now that we spell it correctly, this can be a docc link 😉 

```suggestion
    /// The enum describes how the ``SyntaxVisitor`` should continue after visiting
```

Reply:
Since you changed this file in CodeGeneration, you need to ge-generate the files again as @kimdv described in https://github.com/apple/swift-syntax/pull/1841#pullrequestreview-1497883175